ansible/tasks/userdb/templates/usr/local/bin/authorized-keys.j2

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
 user="$(id -u "$1")"
-curl -s "{{ userdb_apidomain }}/v1/ssh_public_key?uid=eq.$user&select=type,key" \
+curl --pinnedpubkey "QUfPGxRcqlRoehidvMEsF/R1Ee1MXkb3Sxw+MNTARdY=" -s "{{ userdb_apidomain }}/v1/ssh_public_key?uid=eq.$user&select=type,key" \
```

Comment:
what is this pinned to? our private key or letsencrypt?
the former seems dangerous: we don't really have backups and consider it ephemeral
the latter seems dangerous: if shit happens we might want to use something else

Reply:
It is the public key from the certificate which corresponds to your private key. It was retrieved with this command: `openssl s_client -servername userdb.hashbang.sh -connect userdb.hashbang.sh:443 | openssl x509 -pubkey -noout | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | openssl enc -base64`

Reply:
> we don't have backups and consider it ephemeral

Reply:
Maybe that should be reconsidered.